webapp/src/containers/modals/SubscribeModal/index.tsx

```diff
@@ -118,15 +119,30 @@ const SubscribeModal = () => {
         }
     };
 
+    const setSelectedDropdownOption = (field: SubscriptionModalFields, newValue: string, selectedOption?: Record<string, string>) => {
+        onChangeFormField(field as SubscriptionModalFields, newValue);
+
+        if (field === 'project' && selectedOption) {
```

Comment:
Do we need to create a constant for this?

Reply:
no we are using string literal as per the type, however we have added a TODO to create enums in future